backend/dataall/modules/mlstudio/db/models.py

```diff
@@ -19,20 +22,21 @@ class SagemakerStudio(Resource, Base):
     userRoleForSagemakerStudio = query_expression()
 
 
-class SagemakerStudioUserProfile(Resource, Base):
+class SagemakerStudioUser(Resource, Base):
+    """Describes ORM model for sagemaker ML Studio user"""
     __tablename__ = 'sagemaker_studio_user_profile'
     environmentUri = Column(String, ForeignKey("environment.environmentUri"), nullable=False)
-    sagemakerStudioUserProfileUri = Column(
+    sagemakerStudioUserUri = Column(
```

Comment:
Shouldn't we create a migration script if we are renaming the columns in the database? On the other hand, we could provide column name

Reply:
It is part of `4a0618805341_rename_sgm_studio_permissions.py`